backend/api/migrations/0054_alter_function_status.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 4.2.3 on 2023-09-27 16:09
+
+from django.db import migrations
+from django.db import models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("api", "0053_function_status"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="function",
+            name="status",
+            field=models.CharField(
+                choices=[
+                    ("FUNCTION_STATUS_UNKNOWN", "Function Status Unknown"),
+                    ("FUNCTION_STATUS_WAITING", "Function Status Waiting"),
+                    ("FUNCTION_STATUS_BUILDING", "Function Status Building"),
+                    ("FUNCTION_STATUS_READY", "Function Status Ready"),
+                    ("FUNCTION_STATUS_CANCELED", "Function Status Canceled"),
+                    ("FUNCTION_STATUS_FAILED", "Function Status Failed"),
+                ],
+                max_length=64,
```

Comment:
THis migration may need a data migration for the (non existing) case where someone has applied the previous migration but not this one. Alternatively, we could merge these two migrations.